Core/include/Acts/Seeding/Seedfinder.ipp

```diff
@@ -353,12 +353,12 @@ void Seedfinder<external_spacepoint_t, platform_t>::createSeedsForGroup(
           }
 
           // correcting other variables
-          float xB = rBTransf[0] - rTTransf[0];
-          float yB = rBTransf[1] - rTTransf[1];
-          float zB = rBTransf[2] - rTTransf[2];
-          float xT = rBTransf[0] - rTTransf[0];
-          float yT = rBTransf[1] - rTTransf[1];
-          float zT = rBTransf[2] - rTTransf[2];
+          float xB = rBTransf[0] - rMTransf[0];
+          float yB = rBTransf[1] - rMTransf[1];
+          float zB = rBTransf[2] - rMTransf[2];
+          float xT = rTTransf[0] - rMTransf[0];
+          float yT = rTTransf[1] - rMTransf[1];
+          float zT = rTTransf[2] - rMTransf[2];
```

Comment:
This is hard to understand (for me at least), so I'm not sure if this fixes the problem :)
should this not be `M - B` instead of `B - M`? 
can you maybe add a more expressive comment? Thank you!